### ISSUE TITLE
Question: how to calculate padding automatically?

### ISSUE THREAD

**Issue by alexey-milovidov** [leeoniya/uPlot#711]
I do this:
```
padding: [ null, null, null, (Math.ceil(Math.log10(Math.max(...json[1]))) + Math.floor(Math.log10(Math.max(...json[1])) / 3)) * 6 ],
```

But it does not feel natural.

**Comment by leeoniya** [comment]
can you elaborate on the use case?

i'm assuming you're trying to get something like this: https://leeoniya.github.io/uPlot/demos/axis-autosize.html

(related to rendering long tick values in https://github.com/leeoniya/uPlot/issues/713)

**Comment by alexey-milovidov** [comment]
Yes, thank you for pointing to this example. Looks like exactly what I need!
